portsyncd/sample/interfaces

```diff
@@ -0,0 +1,8 @@
+# interfaces file
+#
+# The main configuration file that is located in folder /etc/network/.
+
+auto Ethernet0
+iface Ethernet0 inet static
+    address 10.0.0.0
+    netmask 255.255.255.254
```

Comment:
can you add Ethernet4 in this example? as it is referenced in the vlan_interfaces.